realm/src/main/java/io/realm/Realm.java

```diff
@@ -1138,6 +1372,7 @@ boolean contains(Class<?> clazz) {
      * @return A sorted RealmResults containing the objects.
      * @throws java.lang.IllegalArgumentException if a field name does not exist.
      */
+    @SuppressWarnings("unchecked")
```

Comment:
Even though they are private they are tested intensively through the copyOrUpdate unit tests, so it should be OK to mark them as unchecked.